notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb

```diff
@@ -0,0 +1,10919 @@
+{
```

Comment:
<p>"simply training a new model" =&gt; "simply train a new model"</p><p>"walks you through how to use" =&gt; "walks you through using"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>